config/opendatahub-io/org.yaml

```diff
@@ -284,6 +284,7 @@ orgs:
           caikit: admin
           caikit-nlp: admin
           caikit-tgis-backend: admin
+          caikit-tgis-client: admin
```

Comment:
This should have been `caikit-nlp-client` https://github.com/opendatahub-io/caikit-nlp-client